frontend/src/components/ShootDetails/ShootLifecycleCard.vue

```diff
@@ -16,7 +16,13 @@ SPDX-License-Identifier: Apache-2.0
         </v-list-item-icon>
         <v-list-item-content>
           <v-list-item-title>Hibernation</v-list-item-title>
-          <v-list-item-subtitle class="d-flex align-center pt-1">
+          <v-list-item-subtitle class="d-flex pt-1">
+            <hibernation-constraint-warning
+              :isHibernationPossible="isHibernationPossible"
```

Comment:
html properties are should be `kebab-case`

Reply:
If the name of the component already starts with the word `hibernation` do not repeat it again ana again in the property names. It is clear that everything is related to shoot hibernation.